src/components/CustomChart/CustomChartList.tsx

```diff
@@ -134,9 +135,16 @@ export default function CustomChartList() {
         return (
             <GenericEmptyState
                 image={emptyCustomChart}
-                title="Use custom charts in applications"
-                subTitle={`Import custom charts to use them in apps instead of the default system template.&nbsp;
-          ${renderLearnMoreLink()}`}
+                title={EMPTY_STATE_STATUS.CUSTOM_CHART_LIST.TITLE}
+                subTitle={
+                    <>
+                        Import custom charts to use them in apps instead of the default system template.
+                        <div>
```

Comment:
Remove unecessary div

Reply:
Here <div> is used to make the link appear in next line.